testcode/lineargraph.cpp

```diff
@@ -220,11 +220,11 @@ int main(int argc, char* argv[])
                                  *partition.face_d_td,
                                  *partition.face_d_td_d);
 
-    std::vector<mfem::DenseMatrix> local_edge_traces(num_partitions - 1);
-    std::vector<mfem::DenseMatrix> local_spectral_vertex_targets(num_partitions);
+    std::vector<mfem::DenseMatrix> local_edge_traces;
+    std::vector<mfem::DenseMatrix> local_spectral_vertex_targets;
 
     LocalMixedGraphSpectralTargets localtargets(
-        spect_tol, max_evects, graph.GetM(), graph.GetD(), graph_topology);
+        spect_tol, max_evects, 1, graph.GetM(), graph.GetD(), graph_topology);
```

Comment:
I think it would be more clear to name the variable so that it isn't necessary to look up what `1` does.

```c++
int trace_method = 1;
LocalMixedGraphSpectralTargets localtargets(spect_tol, max_evects, trace_method, graph.GetM(), graph.GetD(), graph_topology);
```